### ISSUE TITLE
Unknown data prediction

### ISSUE THREAD

**Issue by bishwamittra** [alexeyignatiev/minds#1]
Hi, 

This is a nice work. Thanks for publishing the source code. 

I am trying the recent two stage approach for computing decision sets. I am interested in using this decision set to predict new data. 

Is there a function similar to "predict" that I can use to get labels of unknown data for the learned decision sets? 

Thanks, 
Bishwa

**Comment by alexeyignatiev** [comment]
@bishwamittra, good point, thank you. There is no such function at this point but there should be. I will look into this.

**Comment by bishwamittra** [comment]
Thank you very much, Alexey.
